spec/models/concerns/courses/edit_options/qualifications_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe Courses::EditOptions::QualificationConcern do
+  let(:example_model) do
+    klass = Class.new do
+      include Courses::EditOptions::QualificationConcern
+      attr_accessor :level
+    end
+
+    klass.new
+  end
+
+  before do
+    example_model.level = level_value
+  end
+
+  context 'for a course that’s not further education' do
+    let(:level_value) { :primary }
+
+    it 'returns only QTS options for users to choose between' do
+      expect(example_model.qualification_options).to eq(%w[qts pgce_with_qts pgde_with_qts])
+      example_model.qualification_options.each do |q|
+        expect(q.include?('qts')).to be_truthy
```

Comment:
```suggestion
        expect(q).to include('qts')
```

Reply:
Ugh. I just realised that these came out of `spec/models/concerns/courses/edit_options_spec.rb`. Sorry, I didn't catch on when you mentioned that these are being broken out from another spec file. This fix isn't so important then, as they're just being refactored out.